easybuild/easyconfigs/p/PyTorch/PyTorch-1.0.1-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,241 @@
+    '02d278559609982f2dbd973f265caf674274ac7a61ddd867c4956176252d940e',  # NNPACK-20180903.tar.gz
+    'f822489cbbf3cd09db578ea7ac9af40a9d3184f8ff95cb98e8b33b71ed5d89f8',  # sleef-20180509.tar.gz
+    '13100c3deed300bbf16f87d8af3539f432462bfef9d38f0c7e3e387dc2e88676',  # PeachPy-20180219.tar.gz
+    'dc0c9ca760bb1bf858dd1aebaf1471b8437c1b2ef9a9b6acaf302fa14b04194f',  # QNNPACK-20181205.tar.gz
+    'd2dabcd172db8fb91dfa75422a5eb6f55f55f22d047d9f1b02eb6a3be88a09cd',  # fbgemm-20190123.tar.gz
+    '0de43b6eaddd356f1d6cd164f73f37faf2f6c96fd684e1f7ea543ce49c1d144e',  # benchmark-20180606.tar.gz
+    'fd9e9f0d6ef407a162f5f283c52d3c271efc1092377274abe1e30dff83dbc1a4',  # eigen-20180908.tar.gz
+    'e12e096e986221d96d55185e870397b3717b625d6c5d155fbd9a7be314bed06a',  # gemmlowp-20180828.tar.gz
+    '72b4835362d8586e315bdf5c30eea99f2efcf14f19162f00ac0675e8b1b22e61',  # nccl-20181109.tar.gz
+    '7224b03af4acbc54525105bb42e3ecd75c66a3a5b47e8a725ab008467c4109f9',  # cub-20170829.tar.gz
+    'c7418025bd57b3143e651433d0874a39629fce6cc96b43e3f8d5861a8ad23ffb',  # ideep-20181224.tar.gz
+    'f98d2c297353acfab1ecff2054bee65efd2367b4c621e762cf26f50e037a9728',  # mkl-dnn-20190131.tar.gz
+    'f00dc3b142a5be399bdeebd7e7ea369545a35d4fb84c86f98b6b048d72685295',  # mklml_lnx_2019.0.1.20180928.tgz
+    'c8dabee9fd710976c6073a81d2cad2735f95c063819cf8effc001e1940f63283',  # PyTorch-1.0.1-fix-findAVX.patch
+    '569333e194d6a1d716972e17ad2ca1cd72e92a8845cee2f02ab2d5f541f013e9',  # PyTorch-1.0.1-fix-for-intel.patch
+]
+
+builddependencies = [('CMake', '3.12.1')]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('PyYAML', '3.13', versionsuffix),
+    ('hypothesis', '4.5.0', versionsuffix),
+    ('cuDNN', '7.1.4.18'),
+    ('Ninja', '1.8.2'),
+    ('magma', '2.5.0'),
+]
+
+download_dep_fail = True
+
+# This environment is needed to prevent setup.py from invoking git to determine version.
+prebuildopts = 'PYTORCH_BUILD_VERSION=%(version)s PYTORCH_BUILD_NUMBER=1 VERBOSE=1 '
+prebuildopts += ' LDFLAGS="$LDFLAGS -ldl" CUDNN_LIB_DIR=$EBROOTCUDNN/lib CUDNN_INCLUDE_DIR=$EBROOTCUDNN/include '
+preinstallopts = 'VERBOSE=1 LDFLAGS="$LDFLAGS -ldl" '
+preinstallopts += 'CUDNN_LIB_DIR=$EBROOTCUDNN/lib CUDNN_INCLUDE_DIR=$EBROOTCUDNN/include '
+
+# you can choice here: either give a list of CUDA cc version or tell it All
+# by default it does autodetect of the GPU on the local machine
+# https://en.wikipedia.org/wiki/CUDA#GPUs_supported
+# register error with 5.2, 5.3, 6.2
+prebuildopts += ' TORCH_CUDA_ARCH_LIST="3.0 3.2 3.5 3.7 5.0 6.0 6.1 7.0 7.2"'
+
+runtest = 'export PYTHONPATH=%(builddir)s/%(namelower)s-%(version)s/build/lib.linux-x86_64-%(pyshortver)s:$PYTHONPATH '
+# skip test subsuites that include hanging/failing tests for now
+runtest += '&& python test/run_test.py --exclude autograd cpp_extensions c10d torch --verbose'
+
+options = {'modulename': 'torch'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Comment:
Now for the final touchup, the sanity_check_paths is not needed since the above is the default for a PythonPackage.

You can drop it from the foss/2018b version too.

Reply:
done